Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Drivetrain.java

```diff
@@ -42,6 +49,32 @@ public void initDefaultCommand() {
 		setDefaultCommand(new TeleopDrive());
 	}
 
+	/**
+	 * Sets the left side of the drivetrain to use the talons' pids to run at the
+	 * specified speed.
+	 * 
+	 * @param value
+	 *            The speed to run at
+	 */
+	public void dtLeftPIDDrive(double value) {
+		double setValue = value * Robot.getConst("Units per 100ms", 3413);
```

Comment:
Is "Units per 100ms" related to distance per pulse? If so, it should not have a separate pref. The same pref should be used for both Talon and roboRio based PID.